browser/resources/settings/brave_overrides/page_visibility.ts

```diff
@@ -65,15 +67,16 @@ function getPageVisibility () {
   return new Proxy(staticProps, {
     get: function(target, prop) {
       if (prop in target) {
-        return target[prop]
+        return target[prop as keyof Object]
       }
       // default to allow, like chromium
       return true
     }
   })
 }
-// Provide an export in case our overrides want to explicitly import this override
-// Even though we are modifying chromium's override, the es module eval timing may
-// result in the unoverriden value being obtained.
+
+// Provide an export in case our overrides want to explicitly import this
+// override. Even though we are modifying chromium's override, the es module
+// eval timing may result in the unoverridden value being obtained.
 export const pageVisibility = getPageVisibility()
-resetPageVisibilityForTesting(pageVisibility)
+resetPageVisibilityForTesting(pageVisibility as PageVisibility)
```

Comment:
just a comment about the previous code, but its weird to be calling a `ForTesting` method when this is imported in prod

Reply:
I assumed we're doing that to work around the need for patching? I'm not sure, though.